sxw.c

```diff
@@ -82,42 +82,21 @@ extern SW_MARKOV SW_Markov;
 extern Bool prepare_IterationSummary;
 extern Bool storeAllIterations;
 
-
-
 /*************** Module/Local Variable Declarations ***************/
 /***********************************************************/
 /* these are initialized and maybe populated here but are used
  * in sxw_resource.c so they aren't declared static.
  */
-/* ----- 3d arrays ------- */
-RealD * _rootsXphen, /* relative roots X phen in each lyr,grp,pd */
-      * _roots_active, /* "active" in terms of size and phenology */
-      * _roots_active_rel;
-
-
-/* ----- 2D arrays ------- */
-/* malloc'ed here for consistency but only used */
-/* in sxw_resource.c and sxw_soilwat.c */
-
-     /* rgroup by layer, ie, group-level values */
-RealD * _roots_max,     /* read from root distr. file */
-      * _roots_active_sum, /* used in sxw_resource */
-
-     /* rgroup by period */
-      * _phen;          /* phenology read from file */
-
-/* simple vectors hold the resource information for each group */
-/* curr/equ gives the available/required ratio */
-RealF *_resource_cur; /*[MAX_RGROUPS];*/  /* current resource availability for each STEPPE functional type */
-
 // Window of transpiration used by _transp_contribution_by_group() in sxw_resource.c
 // "Window" refers to the number of years over which transpiration data is averaged.
-transp_t transp_window;
-
-// Amount of additional transpiration added for the current year
-RealF added_transp;
-
+transp_t* transp_window;
 pcg32_random_t resource_rng; //rng for swx_resource.c functions.
+SXW_resourceType* SXWResources;
+
+#ifdef SXW_BYMAXSIZE
+/* addition to meet changes specified at the top of the file */
```

Comment:
I investigated the variable defined in this block, `_Grp_BMass`, and I can't find it anywhere in the code. The only other reference in the code is at the top of sxw.c explaining why it exists. Apparently this feature is deprecated, so I'll remove it entirely.

One side note: While this block appears to be new code it is not. The only reason it is showing up as "new" is because I moved a lot of code that was above it. Git is fairly good at recognizing things like this, but in this case it failed to associate new line 97 with it's previous location.